internal/catalogd/webhook/cluster_catalog_webhook.go

```diff
@@ -13,8 +13,6 @@ import (
 
 // +kubebuilder:webhook:admissionReviewVersions={v1},failurePolicy=Fail,groups=olm.operatorframework.io,mutating=true,name=inject-metadata-name.olm.operatorframework.io,path=/mutate-olm-operatorframework-io-v1-clustercatalog,resources=clustercatalogs,verbs=create;update,versions=v1,sideEffects=None,timeoutSeconds=10
 
-// +kubebuilder:rbac:groups=olm.operatorframework.io,resources=clustercatalogs,verbs=get;list;watch;patch;update
```

Comment:
That is interesting, we fail to get the RBAC but not to generate the webhook config based on the markers?

Reply:
`controller-gen` also has a limitation that it cannot use multiple directories (as discovered in my earlier merging of the API directories).

So, I pulled all the RBAC into one place.
The webhook generation still looks at this one directory.